docs/DeveloperGuide.md

```diff
@@ -954,16 +957,16 @@ Steps 3a1-3a2 are repeated until the command entered is correct.
 
 Use case resumes from Step 4.
 
-**Use case 11: Add a student to a lesson**
+**Use case 11: Add student(s) to lesson(s)**
 
-Pre-conditions: The student has been added to TutorAid and the lesson has been created.
+Pre-conditions: The students have been added to TutorAid and the lessons have been created.
 
 **MSS**
 
 1. Tutor requests the list of students and lessons.
 2. TutorAid displays the list of all students and lessons.
-3. Tutor identifies the indices of the specific student and the specific lesson to which they wish to add the student to. The Tutor then requests to add this student to this lesson by providing the indices.   
-4. TutorAid adds the student to the lesson and displays a message to indicate that it has been done.
+3. Tutor identifies the student indices of the specific students and the lesson indices of the specific lessons to which they wish to add these students to. The Tutor then requests to add these students to these lessons by providing the indices.   
```

Comment:
Could you please help to change `indices` to `indexes`? I think can also remove the line break in the first sentence.

Reply:
The line break issue can also be applied to some of the use cases below

Reply:
Perhaps what you mean is that I should leave everything on one line instead of using line break to wrap? If that's what you mean then I am not using any line break.